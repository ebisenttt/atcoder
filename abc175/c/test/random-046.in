156155223086492 705414837354969 480603835909965
