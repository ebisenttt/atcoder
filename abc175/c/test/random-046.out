324448612823473
